src/clr-core/common/utils/global.ts

```diff
@@ -50,20 +54,28 @@ function getVersion() {
     userAgent: navigator.userAgent,
   };
 
-  console.log(JSON.stringify(log, null, 2));
+  if (console && console.log) {
+    console.log(JSON.stringify(log, null, 2));
+  }
   return log;
 }
 
 function getAngularVersion() {
-  const appRoot = document.querySelector('[ng-version]');
+  const appRoot = document && document.querySelector('[ng-version]');
   return appRoot ? appRoot.getAttribute('ng-version') : undefined;
```

Comment:
Since all this code is guarded by `isBrowser()` here https://github.com/vmware/clarity/blob/f4c2610acee7d37407c1cbe0dc4d125b7013721e/src/clr-core/common/utils/global.ts#L29 do we need have the window/document checks? Or could they all use the same `isBrowser()` function?

Reply:
We might be able to use the isBrowser function. But I was sledgehammering it because I'm tired of fighting SSR. I'm willing to give isBrowser a shot. But I don't really want to have to come back and fix this over again.

Reply:
Note also that this issue comes from Stackblitz and Vuepress work.

All of this works in ng-universal. Even before the PR...

Reply:
Ok, no worries on the SSR then, if we can get the stackblitz issue in then we can go ahead get the PR merged. I can block off time to do a in-depth check with core and a handfull of SSR/node setups. Thanks for fixing this!